nist/int.go

```diff
@@ -299,10 +298,16 @@ func (i *Int) MarshalSize() int {
 func (i *Int) MarshalBinary() ([]byte, error) {
 	l := i.MarshalSize()
 	b := i.V.Bytes() // may be shorter than l
-	if ofs := l - len(b); ofs != 0 {
+	offset := l - len(b)
+
+	if i.Bo == LittleEndian {
+		return i.LittleEndian(l, l), nil
```

Comment:
I also think it's not necessary now, but I understood you wanted to keep them, I again may have misunderstood. For me `LittleEndian` &  `BigEndian`  should be removed, does that seem OK ?